pkg/manifest/manifest.go

```diff
@@ -171,10 +171,13 @@ func checkResourceEnablement(annotations map[string]string, capabilities *config
 	unknownCaps := make([]string, 0, numCaps)
 	disabledCaps := make([]string, 0, numCaps)
 
-	var isKnownCap bool = false
-	var isEnabledCap bool = false
+	var isKnownCap bool
+	var isEnabledCap bool
```

Comment:
nit: `is*Cap` are only used within the loop, so you can shift the whole `var ...` or `isKnownCap := false` or whatever into the loop and drop the `var` declarations here.